ding/reward_model/rnd_reward_model.py

```diff
@@ -156,3 +169,14 @@ def collect_data(self, data: list) -> None:
 
     def clear_data(self) -> None:
         self.train_obs.clear()
+
+    def reward_deepcopy(self, train_data):
```

Comment:
you can move the default implementation in basic class, and derived class can modify or overwrite it when it needs